docs/user/convert/function_usage.rst

```diff
@@ -541,7 +541,7 @@ ID Functions
 stringToBytes
 ^^^^^^^^^^^^^
 
-Description: Converts a string to a UTF-8 byte array (to pass to ``md5()`` or ``base64()``).
+Description: Converts a string to a UTF-8 byte array (to pass to other functions like ``md5()``).
```

Comment:
Hmm.... do we want to continue mentioning base64Encode?  (I'm fine either way, just double-checking.)

Reply:
i mainly removed this b/c i'm not sure why you would take a string, get its bytes, then base64 encode them? but if you think that's a normal use case i'll add it back in.

Reply:
Nevermind; you are right; don't mind the reviewer asking to keep everything....